sphinx/users/vaa3d/index.rst

```diff
@@ -1,8 +1,8 @@
 Vaa3D
 =====
 
-`Vaa3D <https://alleninstitute.org/what-we-do/brain-science/research/products-tools/vaa3d/>`_,
-developed by the `Peng Lab <http://home.penglab.com/>`_ at the `HHMI
+`Vaa3D <https://www.janelia.org/publication/vaa3d-enables-real-time-3d-visualization-and-quantitative-analysis-large-scale>`_,
+developed by the `Peng Lab <https://www.janelia.org/peng-lab>`_ at the `HHMI
```

Comment:
:+1: to the change on line 4, but https://home.penglab.com/proj/vaa3d/home/index.html appears to be working now. That's definitely a better resource than https://www.janelia.org/peng-lab. Can you try switching that link and see if the build passes?

Reply:
thank you, done in https://github.com/ome/bio-formats-documentation/pull/423/commits/7ccf7317f8a62ee33f730871c3318a251d483459